Header Header
uint8 id
uint8 goal_x
uint8 goal_y
string data